### DISCUSSION TITLE
HTTP POST 2 does not support values of list type

### DISCUSSION THREAD

**Discussion by bingovhh** [jertel/elastalert2#829] [Q&A] [answered]
HTTP POST 2 has no option 'http_post_static_payload', 'http_post2_payload' and 'http_post2_raw_fields' does not support values of list type, can you add this option to support the following configuration? Thanks~
```
http_post2_static_payload:
  list_key:
    - "foo"
    - "bar"
```

**Comment by nsano-rururu** [comment, marked as answer]
There are no plans to support it. Pull requests are welcome.

**Reply by bingovhh** [reply]
Thank you, I found that this pull request https://github.com/jertel/elastalert2/pull/823 has solved this problem.
